src/libraries/BitmapLib.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.21;
+
+/// @title  BitmapLib
+library BitmapLib {
```

Comment:
Added a ticket for writing unit tests just for the library: https://github.com/centrifuge/liquidity-pools/issues/339